frame/transaction-payment/dex-tx-payment/Cargo.toml

```diff
@@ -0,0 +1,61 @@
+[package]
+name = "pallet-dex-tx-payment"
+version = "4.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+license = "Apache-2.0"
+homepage = "https://substrate.io"
+repository = "https://github.com/paritytech/substrate/"
+description = "pallet to manage transaction payments in assets"
+readme = "README.md"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+# Substrate dependencies
+sp-core = { version = "7.0.0", default-features = false, path = "../../../primitives/core" }
+sp-io = { version = "7.0.0", default-features = false, path = "../../../primitives/io" }
+sp-runtime = { version = "7.0.0", default-features = false, path = "../../../primitives/runtime" }
+sp-std = { version = "5.0.0", default-features = false, path = "../../../primitives/std" }
+
+frame-support = { version = "4.0.0-dev", default-features = false, path = "../../support" }
+frame-system = { version = "4.0.0-dev", default-features = false, path = "../../system" }
+pallet-transaction-payment = { version = "4.0.0-dev", default-features = false, path = ".." }
+frame-benchmarking = { version = "4.0.0-dev", default-features = false, path = "../../benchmarking", optional = true }
+
+# Other dependencies
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
+scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
+serde = { version = "1.0.136", optional = true }
+
+[dev-dependencies]
+serde_json = "1.0.85"
+
+sp-storage = { version = "7.0.0", default-features = false, path = "../../../primitives/storage" }
+
+pallet-assets = { version = "4.0.0-dev", path = "../../assets" }
+pallet-balances = { version = "4.0.0-dev", path = "../../balances" }
+pallet-dex = { version = "4.0.0-dev", path = "../../dex" }
+
+[features]
+default = ["std"]
+std = [
+	"scale-info/std",
+	"serde",
+	"codec/std",
+	"sp-std/std",
+	"sp-runtime/std",
+	"frame-support/std",
```

Comment:
suggest list in ascending alphabetical order

Reply:
fixed